### DISCUSSION TITLE
RTC and CAN support for FRDM-MCXN236

### DISCUSSION THREAD

**Discussion by iustmehri** [zephyrproject-rtos/zephyr#78683] [Q&A] [answered]
Can you please add RTC and CAN support to FRDM-MCXN236?

**Comment by DerekSnell** [comment, marked as answer]
Hi @iustmehri ,
The RTC and CAN features are planned for FRDM-MCXN236.  The [FRDM-MCXN947](https://docs.zephyrproject.org/latest/boards/nxp/frdm_mcxn947/doc/index.html) is the superset board in this family.  It is a very similar SOC and board.  That board already has CAN support.  And there is an open PR to add RTC at https://github.com/zephyrproject-rtos/zephyr/pull/77247.  The CAN and RTC peripherals are the same hardware IP on these two SOCs, and use the same drivers and bindings.  So you could use the FRDM-MCXN947 now to get these features.  Or if you are eager, you can re-use these features from FRDM-MCXN947, and add to the FRDM-MCXN236 board.

Best regards

**Reply by iustmehri** [reply]
Thank you so much 🙏🏽
